src/test/java/net/openhft/chronicle/bytes/BytesStoreTest.java

```diff
@@ -23,4 +23,57 @@ public void from2() {
         Bytes ell = Bytes.from("Hello").subBytes(1, 3).bytesForRead();
         assertEquals("ell", ell.toString());
     }
+
+    @Test
+    public void testSubSequenceOnHeap() {
+        final Bytes<?> bytes = Bytes.allocateElasticOnHeap();
+
+        bytes.append("Hello");
+
+        testSubSequence(bytes);
+    }
+
+    @Test
+    public void testSubSequenceDirect() {
+        final Bytes<?> bytes = Bytes.allocateElasticDirect();
+
+        bytes.append("Hello");
+
+        testSubSequence(bytes);
+
+        bytes.releaseLast();
+    }
+
+    private void testSubSequence(Bytes<?> hello) {
+        CharSequence helloSubsequence = hello.subSequence(0, 5);
+        assertEquals("Hello", helloSubsequence.toString());
+
+        CharSequence hellSubsequence = hello.subSequence(0, 4);
+        assertEquals("Hell", hellSubsequence.toString());
+
+        CharSequence ellSubsequence = hello.subSequence(1, 4);
```

Comment:
Brilliant variable name...

Reply:
Didn't want to steal fame - this is Peter's idea from another test method :grinning: